apiary.apib

```diff
@@ -625,6 +625,22 @@ Get details for a single user.
             + function: `Sales` (string)
             + time_zone: `Europe/Brussels` (string)
 
+
+### users.updateRates [POST /users.updateRates]
```

Comment:
We need to keep this PR open until we fully release profit on projects: the internal rate is not visible unless you have the profit FF enabled.